csfle/dotnet/gcp/reader/CSFLE/InsertEncryptedDocument.cs

```diff
@@ -94,9 +94,9 @@ public static void Insert()
             // start-extra-options
             var extraOptions = new Dictionary<string, object>()
             {
-                { "mongocryptdSpawnPath", credentials["MONGOCRYPTD_PATH"]},
+                { cryptSharedLibPath: credentials["SHARED_LIB_PATH"]},
             };
-            // end-extra-options
+    // end-extra-options
```

Comment:
Suggestion:
It may not affect anything, but I think the spaces should match the same indentation as the next code line, which is what it was before this change.

Reply:
Good catch. The autoformatter that runs when this generates is not great